lib/activerecord-import/import.rb

```diff
@@ -476,7 +476,7 @@ def import_with_validations( column_names, array_of_attributes, options = {} )
           next if model.valid?(options[:validate_with_context])
           raise(ActiveRecord::RecordInvalid, model) if options[:raise_error]
           array_of_attributes[i] = nil
-          failed_instances << model.dup
```

Comment:
The reason the model is duped is because it is being reused to validate attribute values instead of instantiating a new instance for every record. I think it might be possible to dupe the errors as well.

Can you try something like:
```
failure = model.dup
failure.errors.send(:initialize_dup, model.errors)
```